JavaScript/Objects and Object Constructors/index.js

```diff
@@ -0,0 +1,40 @@
+function Account (name){
+    let currentAmount = 0;
+    this.name = name
+    this.IBAN = 'GR00010003';
+
+    this.deposit = function(num){
```

Comment:
The deposit, withdraw and getBalance methods must belong to the prototype object.